tools/dtest/tests/add_down_node_bring_up.go

```diff
@@ -0,0 +1,77 @@
+package dtests
+
+import (
+	"fmt"
+
+	"github.com/m3db/m3db/tools/dtest/harness"
+	"github.com/m3db/m3db/tools/dtest/util"
+	"github.com/m3db/m3db/x/m3em/convert"
+	m3emnode "github.com/m3db/m3db/x/m3em/node"
+
+	"github.com/m3db/m3em/node"
+	xclock "github.com/m3db/m3x/clock"
+	"github.com/spf13/cobra"
+)
+
+var (
+	addDownNodeAndBringUpTestCmd = &cobra.Command{
+		Use:   "add_down_node_bring_up",
+		Short: "Run a dtest where a node that is DOWN, is added to the cluster. Node is then brought up.",
+		Long:  "",
+		Example: `
+TODO(prateek): write up`,
```

Comment:
Can we just put a minimal viable example here, perhaps like `dtest -r add_down_node_bring_up -f file.yaml ...whatever_else...`?